Lib/fontTools/ttLib/tables/otTables.py

```diff
@@ -1187,6 +1187,52 @@ def preWrite(self, font):
 		}
 
 
+class LookupList(BaseTable):
+	def table(self):
```

Comment:
given the method name is a noun instead of a verb, I guess you could make this a @property getter